style.css

```diff
@@ -764,3 +772,55 @@ textarea {
     background-position: right top;
   }
 }
+
+/* ------ mobile menu updates ----- */
+.nav-desktop {
+  position: fixed;
+  left: -100%;
+  top: 0;
+  display: flex;
+  flex-direction: column;
+  font-weight: 500;
+  font-size: 1.5em;
+  color: #edecff;
+  background-color: #191765;
+  width: 91vw;
+  height: 100vh;
+  margin: 0;
+  padding-top: 3em;
+  list-style: none;
+}
```

Comment:
- [ ] The menu isn't correctly covering up the whole visible section, at a certain width the rest of the page becomes visible on the right part and if you scroll down you're able to see some elements covering the menu, please fix this issue.
- [ ] Please make sure to also add some space to the X icon that closes the menu, as it is too close to the menu elements at the moment. It should be always in the furthest position top right of the menu.
- [ ] The background doesn't match the template, kindly make sure to follow the Figma design.

![](https://i.imgur.com/aA9F4vx.png)